src/app/components/LiveLabel/index.tsx

```diff
@@ -50,22 +50,33 @@ const LiveLabel = ({
   }
 
   const circle = [
-    [styles.liveLabelCircle, className === 'first-promo' && styles.firstPromo],
+    styles.pulseOuterCircle,
+    className === 'first-promo' && styles.firstPromo,
   ];
 
   return (
     // The id below is a temporary fix for the a11y nested span's bug experienced in TalkBack, refer to the following issue: https://github.com/bbc/simorgh/issues/9652
     // eslint-disable-next-line jsx-a11y/aria-role
     <span id={id} role="text">
-      <span css={circle}>
-        <span css={styles.livelabelPulse} />
-      </span>
-      <span
-        css={styles.liveLabelText}
-        dir={dir}
-        {...(ariaHidden && { 'aria-hidden': 'true' })}
-      >
-        {`${liveLabel} `}
+      <span css={styles.liveLabelContainer}>
+        <span css={circle}>
+          <svg
```

Comment:
Sure you have / will, if not could you check this is sized proportionally when no CSS, changes colour in high contrast mode in Edge on Windows and also changes according to the selected colours in Firefox when you change your colour preferences please

Reply:
Thanks for this @greenc05 , the SVG was gigantic when I disabled the CSS 😅 I've added a width & height value back onto the SVG which will keep it in proportion with the text beside it when/if CSS is disabled (but will remain the correct size according to UX spec when CSS is applied as normal).